app/root.tsx

```
@@ -291,31 +292,33 @@
+          <Meta />
+          <Links />
+          <EnsureThemeApplied />
+        </head>
+        <body className={`h-full bg-gray-100 dark:bg-slate-800`}>
+          <noscript>
+            <iframe
+              title="GtagManager"
+              src="https://www.googletagmanager.com/ns.html?id=GTM-WH4KFG5"
+              height="0"
+              width="0"
+              className={`hidden invisible`}></iframe>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)